mitodl/open-discussions
# Upgrade pillow and replace deprecated constant

#### Pre-Flight checklist


#### What are the relevant tickets?
Replacement for #3779

#### What's this PR do?
Upgrades pillow to 9.4.0 and replaces a deprecated constant (`Image.ANTIALIAS` -> `Image.LANCZOS`)

#### How should this be manually tested?
Go to your profile page (`http://localhost:8063/profile/<username>`) and upload a new profile image, it should work.


#### What GIF best describes this PR or how it makes you feel?
(Optional)
